main/acle.md

```diff
@@ -1426,6 +1493,21 @@ fused-multiply instructions are available in Neon also.
 intrinsics are available. This implies that `__ARM_NEON` and `__ARM_NEON_FP`
 are both nonzero.
 
+#### Scalable Matrix Extension (SME)
+
+The specification for SME is in
+[**Alpha** state](#current-status-and-anticipated-changes) and may
+change or be extended in the future.
+
+`__ARM_FEATURE_SME` is defined to 1 if the FEAT_SME instructions
```

Comment:
I agree it would be good to be consistent about this, but there seems to be some inconsistency in the ACLE already.  In terms of which consistency to aim for: the Arm ARM doesn't use fixed-width font for the feature names AFAICT, so would it be OK if we standardise on not using a fixed-width font in the ACLE too?